Tests/ValetIntegrationTests/SecureEnclaveIntegrationTests.swift

```diff
@@ -199,4 +202,106 @@ class SecureEnclaveIntegrationTests: XCTestCase
             XCTAssertEqual(try otherValet.string(forKey: key), value)
         }
     }
+
+    // MARK: string(forKey:withPrompt:)
+
+    func test_stringForKeyWithPrompt_throwsItemNotFoundForKeyWithNoValue() throws {
+        XCTAssertThrowsError(try valet.string(forKey: key, withPrompt: prompt)) { error in
+            XCTAssertEqual(error as? KeychainError, .itemNotFound)
+        }
+    }
+
+    func test_stringForKeyWithPrompt_retrievesStringForValidKey() throws {
+        try valet.setString(passcode, forKey: key)
+        XCTAssertEqual(passcode, try valet.string(forKey: key, withPrompt: prompt))
+    }
+
+    // MARK: string(forKey:withPrompt:withFallbackTitle:)
+
+    func test_stringForKeyWithPromptWithFallbackTitle_usesCorrectPolicy() throws {
```

Comment:
What is the "correct" policy? Can we include that in the title of this test method?